cmd/drand-cli/cli_test.go

```diff
@@ -1088,3 +1113,107 @@ func TestSharingWithInvalidFlagCombos(t *testing.T) {
 		"--from flag invalid with --reshare - nodes resharing should already have a secret share and group ready to use",
 	)
 }
+
+//nolint:funlen // This is a test
+func TestMemDBBeaconReJoinsNetworkAfterLongStop(t *testing.T) {
+	sch, err := crypto.GetSchemeFromEnv()
+	require.NoError(t, err)
+	beaconID := test.GetBeaconIDFromEnv()
+
+	// How many rounds to generate while the node is stopped.
+	roundsWhileMissing := 80
+	// If we are in short mode, let's run less rounds.
+	if testing.Short() {
+		roundsWhileMissing = 20
+	}
+
+	period := 1
+	n := 4
+	instances := genDrandInstances(t, beaconID, n)
+	memDBNodeID := len(instances) - 1
+
+	t.Setenv("DRAND_SHARE_SECRET", "testtesttestesttesttesttestesttesttesttestesttesttesttestest")
+	for i := 0; i < memDBNodeID; i++ {
+		inst := instances[i]
+		inst.run(t, beaconID)
+	}
+
+	instances[memDBNodeID].runWithStartArgs(t, beaconID, []string{"--db", "memdb"})
+	memDBNode := instances[memDBNodeID]
+
+	done := make(chan error, n)
```

Comment:
I expected this channel to be a `chan bool` when reading the name - perhaps `errs` is a bit more description